src/client/app/reducers/snippet.ts

```diff
@@ -36,6 +36,10 @@ export function reducer(state = initialState, action: SnippetActions | GitHubAct
         case SnippetActionTypes.LOAD_SNIPPETS_SUCCESS:
             return { ...state, snippets: action.payload };
 
+        case SnippetActionTypes.OPEN_IN_PLAYGROUND:
+            AI.trackEvent(action.type);
```

Comment:
Already track the correlation id in `effects/snippet.ts` when the user initiates the action. When the user ultimately imports the snippet when opening the file, in `import.ts`, we read the correlation id from the document settings and log it again, as a "completion" of the playground action.